.github/dependabot.yml

```diff
@@ -0,0 +1,33 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "bundler"  # See documentation for possible values
+    directory: "/"
+    open-pull-requests-limit: 5
+    schedule:
+      # Check Gemfile updates weekly
+      interval: "weekly"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    open-pull-requests-limit: 5
+    schedule:
+      # Check GitHub Action updates weekly
+      interval: "weekly"
+  - package-ecosystem: "gradle"
+    directory: "/"
+    open-pull-requests-limit: 5
+    schedule:
+      # Check version catalog updates weekly
+      interval: "weekly"
+  - package-ecosystem: "npm"
```

Comment:
I don't think we'll have npm packages in this repository, so I think this can be removed